backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -311,6 +311,39 @@ extern "C" {
         num_samples: u32,
     );
 
+    /// This scratch function allocates the necessary amount of data on the GPU for
+    /// the multi-bit PBS on 64-bit inputs into `pbs_buffer`.
```

Comment:
The comment above is not up to date.

Reply:
Fixed it. BTW, most of comments in `cuda_bind.rs` are not up to date. We should decide to remove or rewrite them.